# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/x264-nprocs.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- x264-snapshot-20061024-2245/common/cpu.c.orig	2006-10-24 22:45:02.000000000 +0200
+++ x264-snapshot-20061024-2245/common/cpu.c	2006-10-30 10:33:21.000000000 +0100
@@ -162,6 +162,9 @@
 #if !defined(HAVE_PTHREAD)
     return 1;
 
+#elif defined(__UCLIBC__)
+    return 1;
+
 #elif defined(SYS_LINUX) || defined(WIN32)
     int np;
 #if defined(WIN32)
